packages/@lwc/template-compiler/src/__tests__/fixtures/html-tags/unkown-element-interpolated/actual.html

```diff
@@ -0,0 +1,5 @@
+<template>
+  <div>
+    <unkonwn>this is {myValue}</unkonwn>
```

Comment:
Typo in `unknown` through the PR 

```suggestion
    <unknown>this is {myValue}</unknown>
```

Reply:
Yes - I borrowed that from other tests and the original [github issue](https://github.com/salesforce/lwc/issues/2084).  I assumed this was so that, should an `<unknown>` tag be added to the HTML spec (not impossible 🤷 ), these tests would still cover the intended cases.  Happy to fix the spelling if that's preferred!